src/imio/smartweb/policy/upgrades/upgrades.py

```
@@ -61,3 +61,25 @@ def set_keycloak_login_group(context):
         oidc.allowed_groups = ["iA.Smartweb"]
     else:
         logger.warning("OIDC plugin not found in acl_users; cannot set allowed_groups.")
+
+
+def clean_ts_api_url(context):
```

Ruff (0.14.14)

[warning] Unused function argument: `context`

(ARG001)